src/azul/__init__.py

```diff
@@ -1772,6 +1772,14 @@ def docker_image_gists_path(self) -> Path:
 
     blocked_user_agents_custom_regex_term = 'blocked_user_agents_custom'
 
+    #: The WAF rules whose matching requests will not be logged in the WAF log
+    #  group, nor trip the `waf_blocked` Cloudwatch alarm.
```

Comment:
```suggestion
    #:  group, nor trip the `waf_blocked` Cloudwatch alarm.
```

Reply:
```suggestion
    #  group, nor trip the `waf_blocked` Cloudwatch alarm
```

This is not a complete sentence.